### PR TITLE
get csv by packageName in OperatorChecker

### PR DESCRIPTION
**Pull request by ZhuoxiLi**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: controllers/operatorchecker/operatorchecker_controller.go [hunk 1]
```diff
@@ -114,7 +119,9 @@ func (r *Reconciler) getCSVBySubscription(ctx context.Context, subscriptionInsta
 
 	var matchCSVList = []olmv1alpha1.ClusterServiceVersion{}
 	for _, csv := range csvList.Items {
-		if strings.Contains(csv.Name, subscriptionInstance.Name) {
+		properties := csv.GetAnnotations()["operatorframework.io/properties"]
+		csvPackageName := strings.SplitN(strings.SplitN(properties, "packageName", 2)[1][3:], "\"",2)[0]
```
Comment (ZhuoxiLi) [inline controllers/operatorchecker/operatorchecker_controller.go:123]: `properties` is a string that cannot be converted to a map.

Comment (horis233) [reply controllers/operatorchecker/operatorchecker_controller.go:123]: Could we use `strings.Contains` here?

Comment (ZhuoxiLi) [reply controllers/operatorchecker/operatorchecker_controller.go]: good idea!

**Review by horis233 — APPROVED** [review]
/lgtm

**Pull request merged**